### PR TITLE
Remove X and P

### PR DESCRIPTION
**Pull request by lillian542**
Removes CV observables `qml.X` and `qml.P` (deprecated last release in favour of `qml.QuadX` and `qml.QuadP`)

### CODE REVIEW

File: pennylane/data/attributes/operator/operator.py [hunk 1]
```diff
@@ -170,8 +170,6 @@ def consumes_types(cls) -> FrozenSet[Type[Operator]]:
                 qml.CatState,
                 qml.NumberOperator,
                 qml.TensorN,
-                qml.X,
-                qml.P,
```
Comment (albi3ro) [inline pennylane/data/attributes/operator/operator.py:174]: Looks like `qml.QuadX` and `qml.QuadP` aren't actually in this list. Do we need to add them in?

**Review by timmysilv — APPROVED** [review]

File: pennylane/data/attributes/operator/operator.py (hunk 1 above)
Comment (lillian542) [reply pennylane/data/attributes/operator/operator.py:174]: Ah, good point, thanks!

File: pennylane/data/attributes/operator/operator.py [hunk 2]
```diff
@@ -170,8 +170,6 @@ def consumes_types(cls) -> FrozenSet[Type[Operator]]:
                 qml.CatState,
                 qml.NumberOperator,
                 qml.TensorN,
```
Comment (lillian542) [inline pennylane/data/attributes/operator/operator.py:172]:
```suggestion
                qml.TensorN,
                qml.QuadX,
                qml.QuadP,
```

**Review by albi3ro — APPROVED** [review]
🚀

**Review by mudit2812 — APPROVED** [review]
Thanks! Here's a 🚀

**Pull request merged**